cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -37,6 +38,7 @@ type kubeletFlagsOpts struct {
 	nodeRegOpts              *kubeadmapi.NodeRegistrationOptions
 	pauseImage               string
 	registerTaintsUsingFlags bool
+	criSocket                string
```

Comment:
may need a to-do item to remind us to clear the following code

```golang
	if opts.criSocket != "" {
		kubeletFlags = append(kubeletFlags, kubeadmapi.Arg{Name: "container-runtime-endpoint", Value: opts.criSocket})
	}
```

TODO: remove this field once the feature NodeLocalCRISocket is GA.